api/well-known/README.md

```diff
@@ -0,0 +1,70 @@
+# Discovery APIs
+The APIs described here allow a user of the Veraison to obtain and view meta information about the Veraison deployment. Information about deployment can be queried according to the service e.g. provisioning or verification.
+
+The relevant resource is created using in response to a client `GET`. The queried information is then outputted as shown in the examples below.
+
+## Well-known API: Provisioning service
+Each well-known resource has the following attributes:
+
+* The allowed provisioning media types;
+* The version of the provisioning service.
+
+### Querying information about the provisioning service
+```json
+>> Request:
+  GET /.well-known/veraison/provisioning
+  Host: veraison.example
+  Accept: application/vnd.veraison.provisioning-discovery+json
+
+
+<< Response:
+  HTTP/1.1 200 OK
+  Content-format: application/vnd.veraison.provisioning-discovery+json
+
+{
+  "media-types": [
+    "application/corim-unsigned+cbor; profile=http://arm.com/psa/iot/1",
+    "application/corim-unsigned+cbor; profile=http://enacttrust.com/veraison/1.0.0",
+    "application/corim-unsigned+cbor; profile=http://arm.com/cca/ssd/1"
+  ],
+  "version": "commit-cb11fa0"
+}
+```
```

Comment:
If you do, like below, you get even more syntax highlighting (push the edit comment button to see the actual code):


* Request:
```http
GET /.well-known/veraison/provisioning
Host: veraison.example
Accept: application/vnd.veraison.provisioning-discovery+json
```

* Response:
```http
HTTP/1.1 200 OK
Content-format: application/vnd.veraison.provisioning-discovery+json

 {
   "media-types": [
     "application/corim-unsigned+cbor; profile=http://arm.com/psa/iot/1",
     "application/corim-unsigned+cbor; profile=http://enacttrust.com/veraison/1.0.0",
     "application/corim-unsigned+cbor; profile=http://arm.com/cca/ssd/1"
   ],
   "version": "commit-cb11fa0"
 }
 ```

Reply:
and avoid the red highlight for text that doesn't parse as JSON :-)

Reply:
When using http parsing as has been done above, the red highlight still remains. However using text parsing gets rid of the red highlight